sites/public/src/layouts/application.tsx

```diff
@@ -21,12 +20,83 @@ const Layout = (props) => {
       </div>
 
       <SiteFooter>
-        <FooterNav copyright={t("footer.copyright")}>
-          <Link href="/privacy">{t("pageTitle.privacy")}</Link>
-          <Link href="/disclaimer">{t("pageTitle.disclaimer")}</Link>
-        </FooterNav>
-        <FooterSection className="bg-black" small>
-          <ExygyFooter />
+        <FooterSection sectionClassName="justify-around" small>
+          <div className="text-left">
+            <img
+              className="h-20 w-20 mr-3"
+              src="/images/bahfa-logo.png"
+              alt="Bay Area Housing Finance Authority Logo"
```

Comment:
these may need to be `t()` i.e. translated as well cc @phattya for more thoughts here

Reply:
Thanks for tagging me in and thinking about translations!  I have a PR I'm working on that adds a lot of translations to the site, so I will check all the content, including the footer. If you can just ensure that the EN is covered in general.json, that will be enough.  @lisunshiny @ifranch

Reply:
Oh...I also realized that you may mean the alt text being translated as well. There are a few instances of this in the current code base, but not a lot. Let's add this to the bug bash list and see if we want to tackle it next week.